src/lib/loo.ts

```diff
@@ -64,3 +65,26 @@ export const filterCompressedLooByAppliedFilters = (
 
   return passesAll;
 };
+
+export const fitMapBoundsToUserLocationNeighbours = (
+  userLocation: LatLngLiteral,
+  map: L.Map
+) => {
+  //find neighbours of the user's location and set the map bounds to fit them
```

Comment:
similar comment to the one above about mentioning that the neighbours are map tiles